src/drc/api/tests/test_enkelvoudiginformatieobject.py

```diff
@@ -85,6 +93,9 @@ def test_read(self):
             'formaat': 'some formaat',
             'taal': 'dut',
             'inhoud': f'http://testserver{test_object.inhoud.url}',
-            'vertrouwelijkheidsaanduiding': '',
+            'link': 'http://www.example.com/',
+            'beschrijving': 'some beschrijving',
+            'vertrouwelijkaanduiding': VertrouwelijkheidsAanduiding.openbaar,
```

Comment:
volgens RGBZ is dit een optioneel veld (kardinaliteit 0..1) - door de default in de factory krijg ik hier de indruk dat `openbaar` de default waarde is (wat niet zo is in het model). Misschien beter om de default in de factory weg te halen zodat hier geen verwarring om bestaat?